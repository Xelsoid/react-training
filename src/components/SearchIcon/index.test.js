import React from 'react';
import ComponentExample from './index';

describe('SearchIcon test', () => {
  it('renders correctly', () => {
    const component = shallow(<ComponentExample />);
    expect(component).toMatchSnapshot();
  });
});
