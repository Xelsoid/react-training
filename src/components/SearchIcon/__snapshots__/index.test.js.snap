// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchIcon test renders correctly 1`] = `
<NavLink
  className="search-icon"
  to="/"
  type="button"
>
  <i
    className="fas fa-search fa-2x fa-flip-horizontal"
  />
</NavLink>
`;
